lib/RTCSession.js

```diff
@@ -39,6 +39,8 @@ const C = {
  */
 const holdMediaTypes = [ 'audio', 'video' ];
 
+const RFC2833 = 'RFC2833';
```

Comment:
Do we really need this const? I think it provides no much value and we can get rid of it.

Reply:
@ibc I agree with you. There is no need for this const.

To make it more clear to @chukitow,  do you want to share your suggestion for removing this constant?